### PR TITLE
Made the country dropdown filterable fixes #263

### LINKED ISSUE #263: Add searchable country dropdown
[Filter dropdown](https://www.w3schools.com/bootstrap/bootstrap_filters.asp) as the number of countries grows bigger, this would enhance UX

### PR DESCRIPTION
**Pull request by mathijshenquet**
Could someone check this?

### CODE REVIEW

**Comment by hnykda** [comment]
Please use the Github Reviewers tool.

**Comment by hnykda** [comment]
I like it and it works locally:

image

Unfortunately, there is currently a conflict with master related to #287 - can you please resolve that?

Also, I would like to @wolverdude to review it as he's the one who wrote most of the code.

**Comment by mathijshenquet** [comment]
> Please use the Github Reviewers tool to ask for reviews.

Will do, should I add you?

It should be merge-able now.

File: src/server/static/js/lines.js [hunk 1]
```diff
@@ -86,9 +81,6 @@ function updateStatistics() {
 
   const { population, data } = baseData.regions[selected.region];
 
-  // TODO: This doesn't seem to work for the first time the page load
-  // TODO: or region is selected, therefore the "None" is here, but would
-  // TODO: be good if it got fixed
```
Comment (mathijshenquet) [inline src/server/static/js/lines.js]: This is also fixed with this

Comment (mathijshenquet) [inline src/server/static/js/lines.js]: This is fixed by pr #291

Comment (hnykda) [reply src/server/static/js/lines.js]: Awesome, thanks

**Comment by hnykda** [comment]
I would really like to merge it ASAP, especially we may get in conflicts with https://github.com/epidemics/covid/pull/300 . Let's not wait for Jon. WDYT @mathijshenquet ?

**Comment by mathijshenquet** [comment]
Yes I agree, it might still be good if someone checks it on IE (is that still a thing?). I will rebase in a bit.

**Comment by hnykda** [comment]
I would just give up on those users, it's less than 0.3%...
image


I don't have any IE device unfortunately. I would claim this is improving UX so much that it's worth it even in the case people got it broken on IE.

**Comment by mathijshenquet** [comment]
Rebased, ping @hnykda @jeromezng

**Comment by hnykda** [comment]
Works on staging: http://staging.epidemicforecasting.org/

I am merging it (-> production)

**Pull request merged**